pkg/volume/testing.go

```diff
@@ -227,3 +231,22 @@ func (fd *FakeDeleter) Delete() error {
 func (fd *FakeDeleter) GetPath() string {
 	return fd.path
 }
+
+// FindEmptyDirectoryUsageOnTmpfs finds the expected usage of an empty directory existing on
+// a tmpfs filesystem on this system.
+func FindEmptyDirectoryUsageOnTmpfs() (*resource.Quantity, error) {
+	tmpDir, err := ioutil.TempDir(os.TempDir(), "metrics_du_test")
+	if err != nil {
+		return nil, err
+	}
+	out, err := exec.Command("nice", "-n", "19", "du", "-s", "-B", "1", tmpDir).CombinedOutput()
```

Comment:
@vissh under the debian flavor in GCE, directories in /tmp report 4096 under du. :(